sungjin/app.js

```diff
@@ -0,0 +1,82 @@
+require("dotenv").config(); //환경변수
+
+const express = require("express");
+const cors = require("cors");
+const logger = require("morgan"); // morgan 모듈 추가하기
+const { DataSource } = require("typeorm");
+const appDataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+appDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+const app = express();
+//미들웨어
+app.use(cors());
+app.use(logger("combined"));
+app.get("/ping", function (req, res, next) {
+  res.json({ message: "pong" });
+});
+const port = process.env.PORT;
```

Comment:
코드의 가독성과 유지 보수를 용이하게 하기 위해 다양한 요인이 존재합니다.
(들여쓰기, 줄바꿈, 공백, 주석 등)

- 해당 부분에서는 코드 간 공백을 두어 가독성을 높이고, 유지 및 보수를 더 용이하게 할 수 있을 것 같습니다!
- 코드로 충분히 설명되는 부분은 주석을 추가할 필요가 없을 것 같습니다. 학습 목적의 주석 역시 실제 서비스가 이루어지는 main 브랜치에 병합이 되기 전 반드시 확인하여 삭제하는 습관도 들여주세요!

아래 내용을 참고하여 수정해주세요!
```suggestion
const express = require("express");
const cors = require("cors");
const logger = require("morgan");
const { DataSource } = require("typeorm");

const appDataSource = new DataSource({
  type: process.env.DB_CONNECTION,
  host: process.env.DB_HOST,
  port: process.env.DB_PORT,
  username: process.env.DB_USERNAME,
  password: process.env.DB_PASSWORD,
  database: process.env.DB_DATABASE,
});

appDataSource.initialize().then(() => {
  console.log("Data Source has been initialized!");
});

const app = express();

//미들웨어
app.use(cors());
app.use(logger("combined"));

app.get("/ping", function (req, res, next) {
  res.json({ message: "pong" });
});

const port = process.env.PORT;
```